packages/@react-native-windows/tester/src/js/examples-win/Flyout/FlyoutExample.windows.tsx

```diff
@@ -36,7 +36,6 @@ let placementValues: string[] = [
 ];
 
 class FlyoutExample extends React.Component<{}, IFlyoutExampleState> {
-  // tslint:disable-next-line:no-any
```

Comment:
All of the stale tslint comments from these existing in an old repo were automatically removed.